tests_e2e/pipeline/pipeline.yml

```diff
@@ -106,17 +107,27 @@ jobs:
       - task: AzureKeyVault@2
         displayName: "Fetch connection info"
         inputs:
-          azureSubscription: 'azuremanagement'
-          KeyVaultName: '$(connection_info)'
+          azureSubscription: $(connection_info)
```

Comment:
where does $(connection_info) variable come from? is this something we have set in our az devops pipelines?

Reply:
Yes, it's existing variable and used to pass key_valut name but now it will be service connection

![image](https://github.com/Azure/WALinuxAgent/assets/84482346/4cebc256-2008-4c31-8467-a41f98394176)